WalletWasabi.Fluent/Converters/OpacityToBoolConverter.cs

```diff
@@ -0,0 +1,21 @@
+using Avalonia.Data.Converters;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace WalletWasabi.Fluent.Converters
+{
+	public class OpacityToBoolConverter : IValueConverter
```

Comment:
see previous comment... make static to save need to declare in resources.

Reply:
done